pegasus/sites.v3/code.org/public/project-ideas.md.partial

```diff
@@ -0,0 +1,131 @@
+    <br>
+    <p><a href="https://studio.code.org/docs/gamelab/">Game Lab Documentation</a></p>
+</div>
+<div style="clear: both; margin-bottom: 20px"></div>
+<table>
+    <thead>
+        <tr>
+            <th>Game Lab Project Ideas</th>
+            <th></th>
+            <th></th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><h3><strong>No Sprites Animation Challenge</strong></h3><p>Test your understanding of variables and the draw loop by creating an animated scene without any sprites!</p><img src="/images/athome/no-sprites.gif" style="max-width:50%"/><br><br><p><a href="https://docs.google.com/document/d/1JGmqwOUr-9LgjbbaEqVK8AQsHSQWvyjgHOJ-9CblU6o/edit#heading=h.jornae4vwl7p">More Details</a></p><p><a href="https://docs.google.com/document/d/1PyqSvPvg01usv6uS1lf-E_kLHAf59iWteyALUqOlEgM/edit#">Project Guide</a></p></td>
+            <td><h3><strong>Randomized Image Creator</strong></h3><p>Apply knowledge of user input and the draw loop to create on-demand randomized images.</p><img src="/images/athome/random-image.gif" style="max-width:50%"/><br><br><p><a href="https://docs.google.com/document/d/1QGIvapBacMyvHVRDrLH179DEkjOMUBrv7Zh6OYz_Vyo/edit#">More Details</a></p><p><a href="https://docs.google.com/document/d/1-aSVaGYgQFymtLse8oQg1QXdWohbIYzLysfCFJhlRRk/edit#">Project Guide</a></p></td>
+            <td><h3><strong>Teach a Science Concept</strong></h3><p>Create your own pixel animations and add text to create an interactive presentation about a science concept - or any other topic!</p><img src="/images/athome/science-concept.gif" style="max-width:50%"/><br><br><p><a href="https://docs.google.com/document/d/1gevkecZBDUb49KptVE3IqfU1E8Ov5zjbR73bnETsI8g/edit#">More Details</a></p><p><a href="https://docs.google.com/document/d/1mgpQfAsMGJFZlHQwzX0ePIlI8cx8zejLZb0HOwlN95A/edit#">Project Guide</a></p></td>
+        </tr>
+    </tbody>
+</table>
+<div style="clear: both; margin-bottom: 40px"></div>
+<h2 style="margin-top:10px"><a href="https://code.org/educate/applab"><strong>App Lab</strong></a></h2>
+<div class="col-33" style="padding-right: 20px;">
+    <img src="/images/athome/app-lab.jpg">
+</div>
+<div class="col-66">
+    <p>App Lab is a programming environment where you can make simple apps. Design an app, code with blocks or JavaScript to make it work, then share your app in seconds.  If you have no experience with App Lab you could get up to speed by trying out: <a href="https://studio.code.org/s/applab-intro/lessons/1/levels/1">Hour of Code - Intro to App Lab</a>.  Alternatively, the projects are suitable for students who have completed the 2nd chapter of CS Discoveries  <a href="https://curriculum.code.org/csd/unit4/">Unit 4 - The Design Process</a> or the first chapter of CS Principles <a href="https://curriculum.code.org/csp-19/unit5/">Unit 5 - Building Apps</a>.</p>
```

Comment:
Similar to my previous comment: 
Before: 
`the 2nd chapter of CS Discoveries  <a href="https://curriculum.code.org/csd/unit4/">Unit 4 - The Design Process</a> or the first chapter of CS Principles <a href="https://curriculum.code.org/csp-19/unit5/">Unit 5 - Building Apps</a>`

After:
`The Design Process unit from <a href="https://studio.code.org/courses/csd">CS Discoveries</a> or Intro to App Design from <a href="https://studio.code.org/courses/csp">CS Principles</a> `

I'm thinking we can leave out "Chapter 2" in the CSD recommendation since we just want them to have done the whole unit. 
@dancodedotorg

Reply:
Yah - that feels right. Wonder if we should also recommend the [Self-Paced Introduction to App Lab](https://studio.code.org/s/csp5-virtual) (similar to my Game Lab suggestion)?

Reply:
Yup, good thinking!